### PR TITLE
Fix tag tests

### PR DESCRIPTION
**Pull request by psav**
{{pytest: cfme/tests/configure/test_access_control.py -k "edit_tag or remove_tag" }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 452e0a5649393cc46b97b44bf33e9da85ff0efb5:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/configure/test_access_control.py:20:1: F401 'quoteattr' imported but unused
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 149ba8ea668d1038fc084eae02b1c03b1da1bb0d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by psav** [comment]
This breaks a 5.4 test...group info block is not good.....so maybe I'll do a verpick here

**Comment by dajoRH** [comment]
Lint report for commit f143820b452170c9ea6b4ec4a721ba3693593de3:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_access_control.py [hunk 1]
```diff
@@ -236,9 +232,15 @@ def test_group_edit_tag():
...
-    tag = sel.text(row).strip()
+    e = version.pick({
+        version.LOWEST:
+            lambda: sel.elements(
+                "//*[(self::th or self::td or self::label) and normalize-space(.)={}]/../.."
+                "//td[img[contains(@src, 'smarttag')]]".format(quoteattr("My Company Tags"))
+            ),
+        '5.5': lambda: InfoBlock('Smart Management', 'My Company Tags').element
+    })
+    tag = sel.text(e()).strip()
```
Comment (mfalesni) [inline cfme/tests/configure/test_access_control.py]: Only this one failed, I will take a peek on this after merge.

**Pull request merged**